test/serverEditorAdapter.test.ts

```diff
@@ -88,7 +88,7 @@ suite('ServerEditorAdapter', () => {
 
     suite('showEditor', () => {
         test('check if parse method is called with right params', async () => {
-            const parseStub = sandbox.stub(vscode.Uri, 'parse');
+            const parseStub = sandbox.stub(vscode.Uri, 'parse').resolves(uriDoc);
```

Comment:
Shouldn't it be returns(uriDoc)? vscode.Uri.parse doesn't return any Promise

Reply:
yeah, that sounds right.